zebedee-cms/src/test/java/com/github/onsdigital/zebedee/model/CollectionTest.java

```diff
@@ -1397,6 +1397,50 @@ public void populateReleaseQuietlyShouldAddLinksToReleasePageForCollectionConten
         assertEquals("/some/uri", result.getRelatedDocuments().get(0).getUri().toString());
     }
 
+    @Test
+    public void populateReleaseQuietlyShouldAddLinksToReleasePageForCollectionContentCMD() throws ZebedeeException, IOException {
+        // Given a collection that is associated with a release and has a CMD dataset
+        String uri = String.format("/releases/%s", Random.id());
+        Release release = createRelease(uri, new DateTime().plusWeeks(4).toDate());
+
+        CollectionDescription description = new CollectionDescription();
+        description.setId(Random.id());
+        description.setName(description.getId());
+
+        collection.getDescription().setReleaseUri(uri);
+        collection.associateWithRelease(publisher1Session, release, collectionWriter);
+
+        String releaseJsonUri = uri + "/data.json";
+
+        collection.complete(publisher1Session, releaseJsonUri, recursive);
+        collection.review(publisher2Session, releaseJsonUri, recursive);
+
+        ContentDetail CMDDetail = new ContentDetail("My CMD dataset", "/some/uri", PageType.API_DATASET_LANDING_PAGE);
+        FileUtils.write(collection.getReviewed().getPath().resolve("some/uri/data.json").toFile(),
+                Serialiser.serialise(CMDDetail), Charset.defaultCharset());
+
+
+        // When we attempt to populate the release from the collection.
+
+        FakeCollectionReader collectionReader = new FakeCollectionReader(zebedee.getCollections().getPath().toString(),
+                collection.getDescription().getId());
+        FakeCollectionWriter collectionWriter = new FakeCollectionWriter(zebedee.getCollections().getPath().toString(),
+                collection.getDescription().getId());
+        Iterable<ContentDetail> collectionContent = ContentDetailUtil.resolveDetails(collection.getReviewed(),
+                collectionReader.getReviewed());
+
+        Release result = collection.populateReleaseQuietly(
+                collectionReader,
+                collectionWriter,
+                collectionContent);
+
+        // Then the release is now in progress for the collection and the published flag is set to true
```

Comment:
This comment does not reflect what the code does?

Reply:
Good point. I've changed it now.